cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -431,7 +431,9 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             if (!this.isValidTimeLimit(this.$('.field-time-limit input').val())) {
                 this.$('.field-time-limit input').val('00:30');
             }
-            if (showRulesField) {
+
+            let  showReviewBlock = this.model.get('show_review_rules');
+            if (showRulesField && showReviewBlock) {
```

Comment:
Based on the ACs for the associated ticket, I have a few questions.

1. The ACs want backend-specific waffle flags (i.e. one waffle flag for PSI and one waffle flag for Proctortrack). I see there is only one. I am assuming that is because the value for the flag for PSI is not configurable, therefore there is no reason to have a waffle flag. Is that correct? Should the name of the flag reflect that it's specific to Proctortrack?

2. I don't believe that the conditional logic added here considers the backend that is being used in the course when determining whether to hide the rules field. This will hide the rules field for both PSI and Proctortrack exams if this waffle flag is enabled.